produce/README.md

```diff
@@ -173,6 +177,28 @@ Get a list of all available options using
     --vpn-conf           Disable VPN Configuration
 ```
 
+## Creating Apple Pay merchants and associating them with an App ID
+
+If you want to create a new Apple Pay Merchant Identifier:
+
+    fastlane produce merchant -o merchant.com.example.production -r "Example Merchant Production"
```

Comment:
Agreed, changed. I was copying the existing style of the code samples above for `Enabling / Disabling Application Services`